IBAnimatable/GradientType.swift

```diff
@@ -84,7 +87,7 @@ public enum GradientType: String, IBEnum {
   case redMist
   case tealLove
   case neonLife
-  case manOfSteel
+  case manofSteel
```

Comment:
Yup, straight from the [json](https://raw.githubusercontent.com/Ghosh/uiGradients/master/gradients.json), only thing that was changed was the first charatcer to lowercase